packages/presentation/src/loader/LoaderQueries.tsx

```diff
@@ -53,6 +53,8 @@ export default function LoaderQueries(props: LoaderQueriesProps): JSX.Element {
   useEffect(() => {
     if (channel) {
       const { projectId, dataset } = clientConfig
+      // @todo - Can this be migrated/deprecated in favour of emitting
+      // `presentation/perspective` at a higher level?
```

Comment:
Sure, but we would still need to dispatch it to two channels anyway, right?

Reply:
Yes exactly, and I think we can just pass an array of channel IDs to `send`.

Would need to listen for this event on the other side over `loader/perspective` so left in place to avoid a breaking change for now, but I suppose duplicating wouldn't be a problem.